src/file.ts

```diff
@@ -184,13 +188,12 @@ export class File {
     }
 
     // Find remaining path through globby
-    ;(
-      await globby(globs, {
-        absolute: true,
-        ignore: ['**/node_modules'],
-        ...opts,
-      })
-    ).forEach((p) => filepaths.add(p))
+    const gOpts = { absolute: true, ignore: ['**/node_modules'], ...opts }
+    ;(await globby(globs, gOpts)).forEach((p) => filepaths.add(p))
+
+    for (const cwd of dirs) {
+      ;(await globby('.', { cwd, ...gOpts })).forEach((p) => filepaths.add(p))
```

Comment:
Expanding file patterns won't work in escaped glob path so we are using `cwd` option to specify the base directory.